ran-simulator/files/model/model-7cell-140ue.yaml

```diff
@@ -335,6 +335,6 @@ rrcStateChangesDisabled: false
 initialRrcState: "idle"
 ueCount: 140
 ueCountPerCell: 20
-plmnID: "314628"
+plmnID: "138426"
```

Comment:
Is this changed required for other models? I am trying to understand if this was wrong the whole time? @Nishat-Zaman

Reply:
I just checked the other model so they are correct.